apps/idos-enclave/src/lib/enclave.js

```diff
@@ -171,7 +171,8 @@ export class Enclave {
 
   encrypt(message, receiverPublicKey = this.keyPair.publicKey) {
     const nonce = nacl.randomBytes(nacl.box.nonceLength);
-    const encrypted = nacl.box(message, nonce, receiverPublicKey, this.keyPair.secretKey);
+    const ephemeralKeyPair = nacl.box.keyPair();
```

Comment:
Just realized this has a flaw: the information about the public key is lost, and we'll need it for sure on AGs (since it's not `this.keyPair.publicKey` anymore).